lib/core/cliArgs.js

```diff
@@ -251,7 +251,12 @@ module.exports = (function () {
   }
   // overwrite env var from CLI for logsene-js
   process.LOGSENE_MAX_MESSAGE_FIELD_SIZE = argv.maxLogSize
-
+  if (process.env.HTTP_PROXY !== undefined) {
+    argv.httpProxy = process.env.HTTP_PROXY
```

Comment:
So setting only this arg value is enough to enable proxy, right?

Reply:
yeah, was missing that part to overwrite the CLI paramteter